mission/config/functions.hpp

```diff
@@ -202,9 +202,12 @@ class CfgFunctions
 			class director_init {};
 			class director_job {};
 			class director_check_mission_end {};
+			class director_complete_zone {};
 			class director_open_connected_zones {};
 			class director_open_zone {};
+			// possibly BN specific?
```

Comment:
Need to check whether (a) we need to edit the gameplay director to use closest zones or (b) whether we go back to using connectors again.